tests/management/role/test_view.py

```diff
@@ -649,6 +659,42 @@ def test_list_role_with_additional_fields_success(self):
             self.assertIsNotNone(iterRole.get("groups_in")[0]["uuid"])
             self.assertIsNotNone(iterRole.get("groups_in")[0]["description"])
 
+    def test_list_role_with_groups_in_includes_defaults(self):
+        """Test that we can read a list of roles with the groups_in field correctly set."""
+        field_1 = "groups_in"
+        new_display_fields = self.display_fields
+        new_display_fields.add(field_1)
+
+        # list a role
+        url = "{}?add_fields={}&scope=principal".format(URL, field_1)
+        client = APIClient()
+        response = client.get(url, **self.headers)
+
+        # three parts in response: meta, links and data
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        for keyname in ["meta", "links", "data"]:
+            self.assertIn(keyname, response.data)
+        self.assertIsInstance(response.data.get("data"), list)
+
+        for iterRole in response.data.get("data"):
+            self.assertEqual(new_display_fields, set(iterRole.keys()))
+
+            # a role thats in 2 groups, one is platform default
+            if iterRole.get("name") == "default_role":
```

Comment:
- [ ] need to rework this test: this will only execute the asserts if there are roles with specified names, if they are not returned the test is useless
- [ ] add another test to test both a principal and not principal scoped request